### PR TITLE
Fix LiveTestTopicQueue usage example

### PR DESCRIPTION
**Pull request by epochiero**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,5 @@
 *~
+*pyc
 __pycache__
```
Comment (etoccalino) [inline .gitignore:3]: If you're getting `*.pyc` files tracked by git, in spite of `__pycache__` being ignored, then you are probably using python2... is that correct?

**Comment by etoccalino** [comment]
Completely missed this. I'll review right away!

File: .gitignore (hunk 1 above)
Comment (epochiero) [inline .gitignore:3]: I added it because there was a .pyc file lying around in my working copy,but I'm using python3

Comment (etoccalino) [inline .gitignore:3]: Gottcha. good to know. Thanks for the doc fix!

**Pull request merged**